src/Models/GFlasso.cpp

```diff
@@ -97,15 +97,15 @@ MatrixXd Gflasso::get_X(){
 }
 
 MatrixXd Gflasso::get_Y(){
-    return Y;
+    return y;
 };
 
 MatrixXd Gflasso::get_beta() {
     return beta;
 }
 
 void Gflasso::assertReadyToRun() {
-    throw runtime_error("Gflasso not implemented");
+    //throw runtime_error("Gflasso not implemented");
```

Comment:
It seems like this is using a different training paradigm - no algorithm attached? Is that correct?